viewing_party/party.py

```diff
@@ -19,25 +16,13 @@ def add_to_watched(user_data, movie):
         "watched": [movie] 
     }

```

Comment:
This issue with replacing the key instead of using `append` was pointed out the first time, but since I felt other work showed your knowledge, it's not a big deal. It still would have been preferred to fix it.

The issue exists in `add_to_watchlist` as well.